### PR TITLE
Added more tests for AdminnController

### PR DESCRIPTION
**Pull request by OlehHatsko**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by sooooolo — APPROVED** [review]

**Review by randomVovaKuhot — APPROVED** [review]

File: EPlast/EPlast.Tests/Controllers/AdminControllerTests.cs [hunk 1]
```diff
@@ -169,6 +240,50 @@ public async Task GetAdmins_Valid_Test()
             Assert.IsInstanceOf<OkObjectResult>(result);
         }
 
+        [Test]
+        public async Task GetAdmins_WrongCityId_ReturnBadRequestResult()
```
Comment (vovikboyko) [inline EPlast/EPlast.Tests/Controllers/AdminControllerTests.cs:244]: Use one same convention fo test naming. Use "Test" ending for every or for non tets.

**Review by vovikboyko — CHANGES_REQUESTED** [review]

**Review by vovikboyko — APPROVED** [review]

File: EPlast/EPlast.Tests/Controllers/AdminControllerTests.cs [hunk 2]
```diff
@@ -43,6 +42,62 @@ public AdminControllerTest()
...
+        }
+
+        [Test]
+        public void ChangeCurrentUserRole_UserExists_Test()
+        {
+            //Arrange
+            AdminController adminController = CreateAdminController;
+            
+            //Act
+            var result = adminController.ChangeCurrentUserRole("user", It.IsAny<string>());
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Controllers/AdminControllerTests.cs:93]: Magic string

File: EPlast/EPlast.Tests/Controllers/AdminControllerTests.cs [hunk 3]
```diff
@@ -103,6 +158,22 @@ public async Task Edit_Invalid_Test()
...
         }
 
+        [Test]
+        public async Task Edit_CouldNotFindUser_Test()
+        {
+            //Arrange
+            AdminController adminController = CreateAdminController;
+
+            // Act
+            var result = await adminController.Edit("user");
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Controllers/AdminControllerTests.cs:168]: Magic string

File: EPlast/EPlast.Tests/Controllers/AdminControllerTests.cs [hunk 4]
```diff
@@ -169,6 +240,50 @@ public async Task GetAdmins_Valid_Test()
...
         }
 
+        [Test]
+        public async Task GetAdmins_WrongCityId_Test()
+        {
+            //Arrange
+            AdminController adminController = CreateAdminController;
+
+            //Act
+            var result = await adminController.GetAdmins(0);
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Controllers/AdminControllerTests.cs:250]: Magic number

File: EPlast/EPlast.Tests/Controllers/AdminControllerTests.cs [hunk 5]
```diff
@@ -169,6 +240,50 @@ public async Task GetAdmins_Valid_Test()
...
+        }
+
+        [Test]
+        public async Task GetCityAndRegionAdminsOfUser_UserExists_Test()
+        {
+            //Arrange
+            AdminController adminController = CreateAdminController;
+
+            //Act
+            var result = await adminController.GetCityAndRegionAdminsOfUser("user");
```
Comment (voroniak) [inline EPlast/EPlast.Tests/Controllers/AdminControllerTests.cs:279]: Magic string

**Review by voroniak — CHANGES_REQUESTED** [review]

**Review by voroniak — APPROVED** [review]

**Pull request merged**